rakelib/test.rake

```diff
@@ -24,7 +24,7 @@ namespace "test" do
     # logstash-core-event specs since currently this is the most complete Event and Timestamp specs
     # which actually defines the Event contract and should pass regardless of the actuall underlying
     # implementation.
-    specs = ["spec/**/*_spec.rb", "logstash-core/spec/**/*_spec.rb", "logstash-core-event/spec/**/*_spec.rb"]
+    specs = ["test/unit/**/*_spec.rb"]
```

Comment:
good point with the event specs, I guess this makes sense to have it separate. Somehow it starts to get strange to me this days when we get

- logstash
- logstash-core
- logstash-event
- logstash-event-java

where to have specs relevant to one close to the source of them. I guess a good mid point would be to have the event specs closer to the event folder as I guess we will have this as interchangeable  components with the java one, right? 

I think your point of having event separated might makes sense, I would just dislike having test to spread if not necessary as it makes tracking them harder, at less for me.